### DISCUSSION TITLE
Puma 6.1 codename

### DISCUSSION THREAD

**Discussion by nateberkopec** [puma/puma#3075] [Ideas]
@MSP-Greg you have most commits since 6.0:

```
15  MSP-Greg
     9  Patrik Ragnarsson
     6  Nate Berkopec
     2  Juanito Fatas
     1  Amr El Bakry
     1  Andrey Novikov
     1  Elia Schito
     1  Guillaume Briday
     1  Hiroaki Osawa
     1  James Prior
     1  Jenna Blumenthal
     1  João Marcos
     1  Memuna Haruna
     1  Shayon Mukherjee
     1  Will Koehler
     1  dependabot[bot]
```

...and it's been far too long since you named a release anyway. Would you do us the honor of naming 6.1?

**Comment by MSP-Greg** [comment]
Given that some of those commits were fixes for some of my earlier commits, not sure about that metric...

> Would you do us the honor of naming 6.1?

I would be honored.  Maybe 'The Way Up', the last Pat Metheny Group album?  Not sure if that seems odd or not...
